src/components/AccordionList.jsx

```diff
@@ -4,133 +4,125 @@ import AccordionSummary from "@mui/material/AccordionSummary";
 import AccordionDetails from "@mui/material/AccordionDetails";
 import Typography from "@mui/material/Typography";
 import ExpandMoreIcon from "@mui/icons-material/ExpandMore";
-import { FormattedMessage } from "react-intl";
+import { useIntl } from "react-intl";
 
-export default function SimpleAccordion() {
-  return (
+const AccordionList = () => {
+  const intl = useIntl();
+
+  const AccordionData = [
```

Comment:
🔧 Could you put this into its own file in the same directory.

Reply:
Addressed: commit 9ed5a025e23ab2e934a5c8e03db5fa335555f6e3

Reply:
:wrench: Can you make this a `js` file?